docs/_instructions/export-apiguardian.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+class: Project
+title: -export-apiguardian PACKAGE-SPEC, ( ',' PACKAGE-SPEC )*
+summary: Exports the given packages where the the `@API` annotation is found on contained classes.
+---
+
+**[API Guardian](https://github.com/apiguardian-team/apiguardian)** is a
+
+> Library that provides the `@API` annotation that is used to  annotate public types, methods, constructors, and fields within a  framework or application in order to publish their status and level of  stability and to indicate how they are intended to be used by consumers  of the API.
+
+The heuristic used by **bnd** to determine which packages to export is as follows:
+
+-  the instruction `-export-apiguardian` is required. The value of the instruction is a package specification like most other [instructions](https://bnd.bndtools.org/chapters/820-instructions.html). The minimal configuration is `-export-apiguardian: *` which instructs bnd to scan all classes in all packages with the plugin.
+- the plugin will only export packages which have _any_ class containing the `@API` annotation and where _not all_ such annotations are marked as `@API(status = INTERNAL)`
+
+#### Examples
+
+Export all packages with version `1.2.3`:
+
+```properties
+-export-apiguardian: *;version="1.2.3"
+```
+
+Export only packages `com.acme.foo.*` but not `com.acme.foo.bar.*`
```

Comment:
Export only packages annotated with API whose status is not INTERNAL named `com.acme.foo.*` but not `com.acme.foo.bar.*`.

Reply:
k